OracleEDQ/12.2.1.3.0/EDQServer.env.list

```diff
@@ -0,0 +1,2 @@
+ADMIN_HOST=<EDQ Admin Server Host>
```

Comment:
Can you not provide a default value, e.g. `localhost`?

Reply:
File EDQServer.env.list removed. As it is replaced by command line paramerters.